<launch>
    <arg name="rviz" default="true" />
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find fast_lio)/config/custom_rosconsole.conf"/>

    <rosparam command="load" file="$(find fast_lio)/config/custom.yaml" />
    
    <!-- <param name="point_filter_num" type="int" value="3"/>
    <param name="max_iteration" type="int" value="5" />
    <param name="filter_size_surf" type="double" value="0.2" />
    <param name="filter_size_map" type="double" value="0.05" /> -->
    
    <param name="feature_extract_enable" type="bool" value="0"/>
    <param name="runtime_pos_log_enable" type="bool" value="0" />
    <node pkg="fast_lio" type="fastlio_mapping" name="laserMapping" output="screen" /> 

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fast_lio)/rviz_cfg/lio_node.rviz" />
    </group>

</launch>